src/azure-cli-core/setup.py

```diff
@@ -59,7 +59,7 @@
     'requests~=2.22',
     'six~=1.12',
     'pkginfo>=1.5.0.1',
-    'azure-mgmt-core==1.2.1',
+    'azure-mgmt-core~=1.2',
```

Comment:
We can at least use `azure-mgmt-core~=1.2.1` to get our users unblocked.

Reply:
Mgmt SDKs use:

https://github.com/Azure/azure-sdk-for-python/blob/ab12b048ddf676fe0ccec16b2167117f0609700d/sdk/network/azure-mgmt-network/setup.py#L85

```py
'azure-mgmt-core>=1.2.0,<2.0.0',
```

which is equivalent to

```py
'azure-mgmt-core~=1.2'
```